messages/notify.go

```diff
@@ -11,23 +11,27 @@ import (
 	"github.com/wmnsk/go-m3ua/messages/params"
 )
 
+var ErrMandatoryNotifyStatusPayloadMissing = errors.New("the Status parameter is mandatory on Notify message (RFC3332), but it's missing")
```

Comment:
I'd like to define a generic error for this purpose like this, instead of this specific one.

````
type ErrMandatoryParamMissing struct {
	ParamType uint32
}

func (e ErrMandatoryParamMissing) Error() string {
	fmt.Sprintf("...")
}
```

If agree on this idea, could you create a new file `errors.go` and put this custom error in it? Please also put following errors in it. 
https://github.com/wmnsk/go-m3ua/blob/master/messages/message.go#L160-L165

Reply:
Totally I agree with the opinion. But if I put the `errors.go` on the `messages` directory, there are two delusive files: `error.go` and `errors.go`. Of course, you know the first one is the definition of the M3UA message, and the second one is the project's error definition.

Is there no problem? Personally, I don't mind it :)

Reply:
anyway, my commit (86e89579c0ab1ddd421eba3f684087148b014574) has removed the error handling so is it no longer needed?

Reply:
You're right. It's no longer needed til some validation for mandatory param is implemented.